docs/ontologymodeling/term-labels-naming-conventions.md

```diff
@@ -0,0 +1,24 @@
+# Term labels and Naming Conventions
+
+One can refer to the same breed by multiple names, including names in different languages (e.g. ‘Artois Hound’ and ‘Chien d’Artois’ refer to the same breed). In addition, the same name is often used to refer to different breeds, of different species. For example, “Tibetan” is the name for a breed of cattle, sheep, chicken, pig, goat, etc. However, a VBO term label must be unique, i.e. multiple terms cannot have the same label, even if their names are the same. In this document, we explain how VBO term labels were created to ensure uniqueness and uniformity. 
+
+Breeds from different species often share the same name. For example, “Tibetan” is the name for a breed of cattle, sheep, chicken, pig, goat, etc. In addition, some breeds are commonly called by names that can represent other types of entities. For example “Cyprus” is used to refer to breeds of cattle, cat, etc., but also to the country “Cyprus”. 
```

Comment:
I would recommend perhaps leaving this out because it is repetitive: Breeds from different species often share the same name. For example, “Tibetan” is the name for a breed of cattle, sheep, chicken, pig, goat, etc. In addition,

Reply:
Please take a look at the proposed changes